plugins/ern_v0.39.0+/react-native-screens_v2.10.1+/RNScreensPackagePlugin.java

```diff
@@ -0,0 +1,16 @@
+package com.walmartlabs.ern.container.plugins;
+
+import android.app.Application;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.facebook.react.ReactInstanceManagerBuilder;
+import com.facebook.react.ReactPackage;
+import com.swmansion.rnscreens.RNScreensPackage;
+
+public class RNScreensPackagePlugin implements ReactPlugin {
+    public ReactPackage hook(@NonNull Application application,
+                     @Nullable ReactPluginConfig config) {
+        return new RNScreensPackage();
+    }
+}
```

Comment:
Please insert final newline. Which version of ern did you use to generate this (I think this was already fixed in a later version)?

Reply:
Oops, will add it. I didn't even know the plugins could be generated by ern, this is just a copy-pasta from another plugin with some adjustments 😅

Reply:
Sorry for the confusion, can you please update one more time and rebase your branch? Only _one_ newline character on the last line please.

Current:

```
}<LF>
<LF>
```

Wanted:

```
}<LF>
```

Previously:

```
}
```

And yes, the `ern` command has had the `create-plugin-config` command for quite a while (maybe it's not advertised enough ;)):

```
ern create-plugin-config react-native-screens
```